script/Arcueid/Day 1/01_00_ARC01_2_1.txt

```diff
@@ -61,7 +61,7 @@ I hope he comes back before the noodles get too soggy.
 [sha:890e96a1253b4fa1867232a1c352d230c2add728]{
 -- Page 140, Offset 1522.
 -- 「だめだ、先輩どっか行っちまったらしい」
-"No good, it seems like Senpai has disappeared."
+"Damn, it seems like Senpai has disappeared."
```

Comment:
Original seems more correct to me (no luck, no joy), his search turned up nothing

Reply:
Yeah, but then it should rather be 'Can't find her' or something.

Reply:
I disagree, "No good" is definitely a thing you'd say if you went to a place and someone wasn't there.